ts-sdk/whirlpool/tests/utils/mockRpc.ts

```diff
@@ -103,7 +103,19 @@ const decoders: Record<string, VariableSizeDecoder<string>> = {
   base64: getBase64Decoder(),
 };
 
-async function getAccountData<T>(address: unknown, opts: unknown): Promise<T> {
+interface AccountData {
```

Comment:
Not 100% sure if the AccountData is always exactly in this format. IIRC data depends on the encoding specified and is not always [string, string]. lamports/rentEpoch might be numbers I think (hence why I put generic T).

Instead of adding this interface you could also just do an assertion in `getBalance` to check if `lamports` exists in the object and then return that:

```
    const accountDataForBalance = await getAccountData(
        addressForBalance,
        config.payload.params[1],
      );
      assert("lamports" in accountDataForBalance);
      return getResponseWithContext<T>(accountDataForBalance.lamports);
 ```